src/JustSaying/Fluent/AccountAddressProvider.cs

```diff
@@ -106,10 +109,14 @@ public Uri GetQueueUriByConvention<T>()
     /// <returns>The <see cref="Uri"/> for this queue.</returns>
     public Uri GetQueueUri(string queueName)
     {
-        var hostname = _regionEndpoint.GetEndpointForService("sqs").Hostname;
-        return new UriBuilder("https", hostname)
+        var endpoint = new AmazonSQSEndpointProvider().ResolveEndpoint(new SQSEndpointParameters
```

Comment:
Can we store the provider in a static readonly field, rather than make a new one every time the method is called?

Reply:
Since I've changed the package to 3.7.3 & 3.7.2, It's not necessary, I've rollback to the old code